pkg/urbit/vere/auto.c

```diff
@@ -353,10 +353,26 @@ u3_auto_exit(u3_auto* car_u)
   }
 }
 
-/* u3_auto_info(): print status info.
+/* u3_auto_info(): status info as a (list mass), all drivers.
 */
-void
+u3_noun
 u3_auto_info(u3_auto* car_u)
+{
+  u3_noun lit = u3_nul;
+
+  while ( car_u ) {
+    if ( car_u->io.info_f ) {
+      lit = u3nc(u3_pier_mass(car_u->nam_m, car_u->io.info_f(car_u)), lit);
+    }
+    car_u = car_u->nex_u;
+  }
+  return lit;
```

Comment:
I think we should either flop this so it's produced in driver link-order, or alphabetize by driver name. The former is easier, so I'd just do that.